WPMedia/arc-themes-components
# THEMES-1255 fix: small issues in carousel component

## Ticket

- [THEMES-1255](https://arcpublishing.atlassian.net/browse/THEMES-1255)

## Description

_Information about what you changed for this PR_

## Acceptance Criteria

_copy from ticket_

## Test Steps

1. Checkout branch - `git checkout THEMES-1255`
2. Update dependencies - `npm i`
3. Run Storybook `npm run storybook`
4. Check out the carousel storybook documentation in the show additional controls option.
Or
You can see the storybook preview: 
* Additional control buttons: https://61eeeb27215b5f003acdf18f-rimkniyarp.chromatic.com/?path=/story/components-carousel--show-additional-controls
* Autoplay: https://61eeeb27215b5f003acdf18f-rimkniyarp.chromatic.com/?path=/story/components-carousel--display-autoplay-toggle

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._


## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-1255]: https://arcpublishing.atlassian.net/browse/THEMES-1255?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
